app/src/main/assets/input_data/Comments.csv

```diff
@@ -6,3 +6,8 @@
 5,True,6,Poor human player (amp)
 6,True,4,Robot had note(s) stuck in it
 7,False,,They were mean
+8,True,7,Evaded defense well
```

Comment:
Someone CAN get rid of the "They were mean" comment.  It was there to prove that the "False" would correctly omit importing it.  That wasn't supposed to make it into a check-in (long ago).